src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/sphinx-design.css

```diff
@@ -1,152 +1,215 @@
+  border: 1px solid #656565 !important;
+  border-radius: var(--ast-sphinx-design-border-radius);
+  padding: 8px 12px;
+  text-align: left;
 }
 
-.sd-card-title {
-  font-family: var(--pst-font-family-base-system);
+.bd-content details.sd-dropdown summary.sd-summary-title:hover {
+  background-color: var(--ast-color-hover-card-background) !important;
+  border-color: var(--ast-color-enable-border);
 }
 
-.bd-content .sd-card .sd-card-body,
-.bd-content .sd-card .sd-card-footer,
-.bd-content .sd-card .sd-card-text {
-  background-color: transparent;
+.bd-content
+  details.sd-dropdown
+  summary.sd-card-header
+  + div.sd-summary-content {
+  background-color: var(--ast-color-sphinx-design-background);
+  border-left: none !important;
 }
 
-html[data-theme="dark"] .sd-shadow-sm {
-  box-shadow: 0 0.1rem 1rem rgba(250, 250, 250, 0.6) !important;
+.bd-content
+  details.sd-dropdown
+  .sd-summary-title:active
+  + div.sd-summary-content:active {
+  background-color: var(--ast-color-enable-card-background);
+  border-color: var(--ast-color-enable-border);
 }
 
-html[data-theme="dark"] .sd-card-img-top[src*=".png"] {
-  filter: invert(0.82) brightness(0.8) contrast(1.2);
+.bd-content
+  details.sd-dropdown
+  .sd-summary-title:hover
+  + div.sd-summary-content:hover {
+  background-color: var(--ast-color-hover-card-background);
+  border-color: var(--ast-color-hover-border);
+  border-left: none;
 }
 
-/* Common styles for all screen sizes */
-.sd-card {
-  border-radius: 0;
-  padding: 10px 10px 10px 10px;
-  font-family: var(--pst-font-family-base) !important;
+.bd-content
+  details.sd-dropdown
+  summary.sd-summary-title:focus
+  + div.sd-summary-content:focus {
+  background-color: var(--ast-color-sphinx-design-background);
+  border-radius: var(--ast-sphinx-design-border-radius);
+  border-color: var(--pst-color-border);
+  border-left: none;
+}
+
+details.sd-dropdown summary.sd-card-header + div.sd-summary-content {
+  color: var(--ast-color-sphinx-design-primary);
+  font-family: var(--ast-sphinx-design-font-family);
+  font-size: var(--ast-sphinx-design-font-size);
+}
+
+/**
+  * Sphinx-design tab
+  */
+
+.bd-content .sd-tab-set > input:checked + label {
+  background-color: transparent;
+  font-family: var(--ast-sphinx-design-font-family);
+  padding: 12px 7.5px 12px 7.5px;
+  border-color: transparent transparent var(--ast-tab-border-color) transparent;
+  border-width: 0px 0px 2px 0px;
+  border-style: none none solid none;
+  color: var(--ast-color-sphinx-design-primary);
+  font-size: var(--ast-sphinx-design-font-size);
+}
+
+.bd-content .sd-tab-set > input:not(:checked, :focus-visible) + label {
+  background-color: var(--ast-background-tab-focus);
+  color: var(--ast-color-sphinx-design-primary);
+  border: none;
+  border-color: var(--ast-color-hover-border);
+  border-width: 0px;
+  text-decoration-line: none;
+  font-weight: normal;
+  padding: 12px 7.5px 12px 7.5px;
```

Comment:
Same as above. We may want to avoid decimals, if possible

Reply:
updated the padding